README.md

```diff
@@ -67,6 +67,17 @@ It consists of:
 
 **YOUR DONE**
 
+## SELinux
+
+If you have SELinux installed and running, add `:Z` to all `volumes` entries in `docker-compose.yaml`, e.g.
+
+```
+...
```

Comment:
double quoted one, I would remove the ''' ticks

Reply:
Thanks for the hint.